django_project/frontend/src/containers/MainPage/Metrics/index.tsx

```diff
@@ -193,16 +245,136 @@ const Metrics = () => {
             pdf.addImage(imageDataUrl, 'PNG', 0, 0, pdfWidth, pdfHeight);
         }
         pdf.save('metrics.pdf');
+        setIconsHidden(false);
     }
 
+
+    type Constants = {
+        [key: string]: boolean;
+    };
+    
+    const constants: Constants = {
```

Comment:
Isn't this from a different PR?

Reply:
@dimasciput it's from the previous PR . I think I pushed them at the same time